pkg/api/bzzchunk.go

```diff
@@ -62,6 +105,9 @@ func (s *server) chunkUploadHandler(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	tag.Address = address
```

Comment:
no. we don't need to open any discussion...  but you are right, we should not expose this API for chunks... we should expose them only for files that are split inside bee. @janos WDYS?

Reply:
This line is indeed a questinable, but here we do not have the root address, we do not know which address is the root, or whether the root address exists at all for the chunks. But the tag uses the address only as a reference, whether it is a root hash or any other. Does it make sense to set the address of the first chunk for the same tag here and never to overwrite it?

Reply:
The most sensible thing i can think of is to add the last chunk's address to it. If we use a splitter that is similar to pyramid chunker, the the last chunk that will come out of it will be the root chunk. so this would work out in that case.

Since we dont know much about the final splitter implementation... i just added it like this.